### PR TITLE
Solved issue of Ellipsis menu overlaps on small screen

### LINKED ISSUE #37517: [GlobalStep] For smaller viewports, "Ellipsis menu" button overlaps with header content of task list on "Woocommerce > Home" page.
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

For smaller viewports, "Ellipsis menu" button overlaps with header content of task list on "Woocommerce > Home" page.

### Expected behavior

For smaller viewports, "Ellipsis menu" button should be displayed properly with respect to header content of task list on "Woocommerce > Home" page.

### Actual behavior

For smaller viewports, "Ellipsis menu" button overlaps with header content of task list on "Woocommerce > Home" page.

### Steps to reproduce

1. Create any test site using the JN site or Atomic site.
2. Install and activate all the required plugins.
3. Install and activate WooCommerce 6.9 Beta 2.
4. Go to "Woocommerce > Home" page.
5. Open the browser devtools.
6. Click on the "Responsive Design mode".
7. Change it to any smaller viewports.
8. Observe that for smaller viewports, "Ellipsis menu" button overlaps with header content of task list.

### WordPress Environment

Woocommerce Version :
WooCommerce 7.6.0-beta.2
WordPress version: 6.2

PC:
Windows 10
Chrome (Version 111.0.5563.147)
Firefox (Version 111.0.1)

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

<details>

`
### WordPress Environment ###

WC Version: 7.6.0
REST API Version: ✔ 7.6.0
WC Blocks Version: ✔ 9.8.4
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.2
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 8.0.28
PHP Post Max Size: 2 GB
PHP Time Limit: 1200
PHP Max Input Vars: 6144
cURL Version: 7.86.0
OpenSSL/1.1.1n

SUHOSIN Installed: –
MySQL Version: 10.6.12-MariaDB-log
Max Upload Size: 2 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.6.0
WC Database Prefix: wp_
Total Database Size: 8.56MB
Database Data Size: 5.15MB
Database Index Size: 3.41MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.06MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.11MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 1.52MB + Index: 0.75MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.16MB + Index: 0.17MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.14MB + Index: 0.13MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.08MB + Index: 0.09MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 0.13MB + Index: 0.03MB + Engine InnoDB
wp_postmeta: Data: 1.52MB + Index: 0.77MB + Engine InnoDB
wp_posts: Data: 0.22MB + Index: 0.06MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.16MB + Index: 0.17MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.17MB + Index: 0.23MB + Engine InnoDB
wp_wc_order_stats: Data: 0.06MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpml_mails: Data: 0.11MB + Index: 0.00MB + Engine InnoDB

### Post Type Counts ###

attachment: 24
page: 15
post: 5
product: 18
product_variation: 7
revision: 31
shop_coupon: 2
shop_order: 212
shop_order_refund: 3
wp_global_styles: 2
wp_template_part: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (2) ###

Query Monitor: by John Blackbourn – 3.12.1
WooCommerce: by Automattic – 7.6.0-beta.2

### Inactive Plugins (4) ###

Akismet Anti-Spam: by Automattic – 5.1
Jetpack: by Automattic – 12.0-a.7
WooCommerce Smooth Generator: by Automattic – 1.0.3
WP Mail Logging: by WP Mail Logging Team – 1.11.0

### Dropin Plugins (3) ###

advanced-cache.php: advanced-cache.php
db.php: Query Monitor Database Class (Drop-in)
object-cache.php: Memcached

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: ✔
Order datastore: WC_Order_Data_Store_CPT

### WC Pages ###

Shop base: #7 - /shop/
Cart: #60 - /test-cart/
Checkout: #63 - /test-checkout/
My account: #10 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty-Two
Version: 1.4
Author URL: https://wordpress.org/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
multichannel-marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: block-editor-feature-enabled
minified-js
new-product-management-experience
product-variation-management
settings

Daily Cron: ✔ Next scheduled: 2023-03-31 06:36:23 +00:00
Options: ✔
Notes: 39
Onboarding: completed

### Action Scheduler ###

Complete: 497
Oldest: 2023-03-30 06:38:40 +0000
Newest: 2023-03-30 12:27:02 +0000

Pending: 2
Oldest: 2023-03-30 14:20:49 +0000
Newest: 2023-03-31 06:38:40 +0000

Failed: 2
Oldest: 2023-03-30 06:40:40 +0000
Newest: 2023-03-30 06:43:37 +0000

### Status report information ###

Generated at: 2023-03-30 14:19:54 +00:00
`

</details>

### PR DESCRIPTION
**Pull request by kshojib**
### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes #37517 .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create any test site.
2. Install and activate WooCommerce 6.9 Beta 2.
3. Go to "Woocommerce > Home" page.
4. Open the browser devtools.
5. Click on the "Responsive Design mode".
6. Change it to any smaller viewports.

### CODE REVIEW

**Review by chihsuan — CHANGES_REQUESTED** [review]
Hey @kshojib Thanks you so much for fixing this issue! 👍 

Tested well. However, I think we don't need to add woocommerce/package.json and pnpm.lock files. Could you please remove them? Thank you!

File: plugins/woocommerce/changelog/fix-37517-fix-ellipsis-menu-overlaps [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+37517-fix-ellipsis-menu-overlaps
```
Comment (chihsuan) [inline plugins/woocommerce/changelog/fix-37517-fix-ellipsis-menu-overlaps]:
```suggestion
Fix ellipsis menu overlaps on small screen
```

**Review by chihsuan — DISMISSED** [review]
Thanks for fixing it @kshojib!

Left one suggestion for the changelog message. Others look good. Pre-approving! 👍

**Review by chihsuan — DISMISSED** [review]
LGTM!

**Review by ilyasfoo — APPROVED** [review]
Tested well, thanks folks! 🚢

**Review by ilyasfoo — CHANGES_REQUESTED** [review]
Actually, it seems there's [lint error](https://github.com/woocommerce/woocommerce/actions/runs/4637576333/jobs/8277388073?pr=37583#step:4:628):

```
plugins/woocommerce-admin lint: client/task-lists/progress-header/progress-header.scss
plugins/woocommerce-admin lint:  77:9  ✖  Expected indentation of 2 tabs  indentation
plugins/woocommerce-admin lint:  86:9  ✖  Expected indentation of 2 tabs  indentation
plugins/woocommerce-admin lint:  ELIFECYCLE  Command failed with exit code 2.
```

**Review by ilyasfoo — APPROVED** [review]
Thanks, @kshojib! LGTM

**Pull request merged**